cli/tests/workers/large_message_worker.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020 the Deno authors. All rights reserved. MIT license.
+
+const dataSmall = "";
+const dataLarge = "x".repeat(10 * 1024);
+
+onmessage = function (e): void {
+  for (let i = 0; i <= 10; i++) {
+    if (i % 2 == 0) {
+      postMessage(dataLarge);
+    } else {
+      postMessage(dataSmall);
+    }
+  }
+};
```

Comment:
It's quite unfortunate that worker is needed to test it, there's definitely additional cost stemming from the fact that each worker runs on a separate thread.

Reply:
Currently workers are the easiest to use, as they can send JSON messages that are pulled by their parrent and these messages are easily too big for the shared queue. If the message is too big, the whole shared queue is omitted and also other messages experience performance deterioration.
Another way to overflow the shared queue is to send too many messages, but this proably is less noticeable.